dbt/models/sale/schema.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  - name: sale
+    tables:
+      - name: flag
```

Comment:
**[Nitpick]** I totally forgot, but there's one last piece here! We need to add a model entry for `sale.vw_ias_salesval_upload` so that it shows up in our docs:

```suggestion
sources:
  - name: sale
    tables:
      - name: flag
      
models:
  - name: sale.vw_ias_salesval_upload
    description: '{{ doc("vw_ias_salesval_upload") }}'
```